src/transformers/trainer.py

```diff
@@ -894,9 +894,16 @@ def train(
             num_update_steps_per_epoch = max_steps
 
         delay_optimizer_creation = self.sharded_ddp is not None and self.sharded_ddp != ShardedDDPOption.SIMPLE
+        model = self.model
+        if self.args.ortmodule:
+            from onnxruntime.training import ORTModule
+            logger.info("Converting to ORTModule ....")
+            model = ORTModule(self.model)
+            self.model = model._original_module
```

Comment:
Do we need this line? `self.model` is already equal to `_original_module`, no?

Reply:
that's right, removing that